pkg/controller/networkpolicy/controller_query_test.go

```diff
@@ -0,0 +1,216 @@
+// Copyright 2019 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package networkpolicy
+
+import (
+	v1 "k8s.io/api/core/v1"
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"testing"
+)
+
+// pods represent kubernetes pods for testing proper query results
+var pods = []v1.Pod{
+	{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "podA",
+			Namespace: "nsA",
+			Labels:    map[string]string{"group": "appliedTo"},
```

Comment:
I'm nitpicking, but I would suggest using some more "traditional" labels for your tests. Maybe `app=foo` or something like this.